dags/bqetl_gud.py

```diff
@@ -0,0 +1,145 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "owner": "jklukas@mozilla.com",
+    "start_date": datetime.datetime(2019, 7, 25, 0, 0),
+    "email": ["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=300),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 1,
+}
+
+with DAG("bqetl_gud", default_args=default_args, schedule_interval="0 1 * * *") as dag:
```

Comment:
The schedule interval changes for new profile smoot queries here (compared to https://github.com/mozilla/telemetry-airflow/blob/6c7a407dec5b3f4a7e7044ebb8e06750c535789f/dags/kpi_dashboard.py#L23). I don't think it makes a difference though.

Reply:
Agreed. It's fine for this to run whenever. The later schedule was implicit dependency management, to make sure we wait for the fxa tables. Not relevant here.